src/mlang/backend_compilers/decoupledExpr.ml

```diff
@@ -308,7 +308,7 @@ let comp op (e1 : constr) (e2 : constr) (stacks : local_stacks)
         if String.equal op "==" && v1 = v2 then Dtrue else Dbinop (op, e1, e2)
     | _ -> Dbinop (op, e1, e2)
   in
-  let e =
+  let _e =
```

Comment:
The definition of `_e` should be deleted since it's no longer useful.

Reply:
And so is `comp` above. This removes the few optimisations made in this constructor, although I'm not sure it actually triggers on the codebase anyways.

Reply:
If we want to stay as close as possible to the old generator, perhaps keeping the optimisation would be better.

Reply:
The new old, I don't think compir has any such optimisation on comparison operators.
This remains to be checked but the optimisations relied on strict equality, so I think they should not interfere with an epsilon approach (at worst the optimisation doesn't trigger although it always will at runtime thanks to the epsilon).

Reply:
Ok. It seems Denis just chose to use the interpreter's epsilon comparison anyway.